Realm/RLMQueryUtil.mm

```diff
@@ -126,6 +126,7 @@ size_t find_first(size_t start, size_t end) const override
         return realm::not_found;
     }
     void set_base_table(const Table*) override {}
+    void verify_column() const override {}
```

Comment:
I ran into this issue as well. We should make sure that an empty override is the right thing to do in this case.